### ISSUE TITLE
Runtime error after compiling heif-convert

### ISSUE THREAD

**Issue by fpereiro** [strukturag/libheif#616]
Hi there! In the first place, thank you for building this awesome library!

We're having trouble running the latest `heif-convert` after cloning the repo and compiling it. All the dependencies have been installed (or so we think), including libde265. The OS is an Ubuntu 20.04.

After successfully running through all the steps, when running `heif-convert` we get the following error: `heif-convert: symbol lookup error: heif-convert: undefined symbol: heif_image_handle_get_auxiliary_type`

Any ideas of which missing library/dependency might be causing this issue?

Many thanks in advance!

**Comment by bradh** [comment]
@fpereiro This is probably an incomplete install (e.g. you have the application calling an old version of the library on your system). Can you try again, or confirm you've since resolved the problem?

**Comment by fpereiro** [comment]
Hi @bradh ! I had completely forgotten I had opened this issue. In the end, we fixed the install by adding a few more dependent libraries. This is what worked for us:

From 
https://github.com/altocodenl/tagaway/blob/6215c4b1cc0a4055dc803d47cb78a3ba0c66005c/utils/provision.sh#L37
all the way to
https://github.com/altocodenl/tagaway/blob/6215c4b1cc0a4055dc803d47cb78a3ba0c66005c/utils/provision.sh#L49

I'll close the issue now. Thank you again for developing this amazing library!
